api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java

```diff
@@ -114,4 +118,17 @@ public static void checkMenuAvailability(CreateOrder cmd) throws
             throw menuNotAvailable(cmd);
```

Comment:
115 line : Extract menu.get().getMenuDateRange() and if statement will be in one line

Reply:
Impossible without addition another if Statement with the same body.
`menu.get().getMenuDateRange()` called after `menu.isPresent()` in one condition.